launcher/image/preload.sh

```diff
@@ -132,8 +132,12 @@ main() {
   # Make sure cache is flushed for the OEM partition.
   sync ${OEM_PATH}
 
+  # Remount as read-only to avoid unexpected changes
```

Comment:
Why do we need this? Are you worried that some other thing is modifying the OEM partition after the fact? Wouldn't that get reflected in the dm-verity?

Reply:
yes, the behavior I saw is the file is all good before the preload.sh finish, but somehow got corrupted before the dm-verity sealing. So dm-verity looks all good, but the files are still corrupted.

Remount it as read-only (as it should be before this script) just to make sure it didn't get written accidently